### PR TITLE
feat: add form input colors and box-shadow to themes

### PR DESCRIPTION
**Pull request by meghaniankov**
# Description

Added input box-shadow and error/focus/default colors to the themes. No change to existing Input styling, for use in Form Input in eevee.

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/themes/bankrate/theme.json [hunk 1]
```diff
@@ -336,6 +337,18 @@
...
+        "color": "grey-90"
+      },
+      "error": {
+        "color": "tomato"
+      },
+      "focus": {
+        "color": "grey-90"
+      },
+      "formInput": {
+        "boxShadow": "inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)"
```
Comment (barneyfoxuk) [inline src/themes/bankrate/theme.json:350]: I would move `inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)` to be in the tokens at the top (under `shadows`) and then use the token name here instead

File: src/themes/journey/theme.json [hunk 2]
```diff
@@ -367,7 +368,17 @@
...
+        "color": "primary"
+      },
+      "default": {
+        "color": "primary"
+      },
+      "error": {
+        "color": "tomato"
+      },
+      "formInput": {
+        "boxShadow": "inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)"
```
Comment (barneyfoxuk) [inline src/themes/journey/theme.json:381]: move to tokens

File: src/themes/save-on-energy/theme.json [hunk 3]
```diff
@@ -303,6 +304,18 @@
           "backgroundColor": "veryLightGrey"
         }
       },
+      "formInput": {
+        "boxShadow": "inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)"
```
Comment (barneyfoxuk) [inline src/themes/save-on-energy/theme.json:308]: move to tokens

File: src/themes/uswitch/theme.json [hunk 4]
```diff
@@ -1050,7 +1051,17 @@
...
+        "color": "primary"
+      },
+      "default": {
+        "color": "primary"
+      },
+      "error": {
+        "color": "tomato"
+      },
+      "formInput": {
+        "boxShadow": "inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)"
```
Comment (barneyfoxuk) [inline src/themes/uswitch/theme.json:1064]: Move to tokens

File: src/themes/bankrate/theme.json (hunk 1 above)
Comment (meghaniankov) [reply src/themes/bankrate/theme.json]: I've updated and moved to `shadows`

File: src/themes/journey/theme.json (hunk 2 above)
Comment (meghaniankov) [reply src/themes/journey/theme.json]: I've updated and moved to `shadows`

**Pull request merged**